include/taskolib/Sequence.h

```diff
@@ -429,22 +428,19 @@ class Sequence
      * \exception Error is thrown if the sequence has no capacity for additional entries
      *            or if it is currently running.
      */
-    ConstIterator insert(ConstIterator iter, const Step& step);
+    template <typename StepType, std::enable_if_t<
+        std::is_same<gul14::remove_cvref_t<StepType>, Step>::value, bool> = true>
```

Comment:
Why do you use this compile time check? Maybe it is off-topic but what is the semantic of `gul14::remove_cvref`? Do you know where I will find in the GUL14 docu? Is it an adaption of `std::remove_cvref`? Is it a combination of `remove_cv` and `remove_ref`? I am a little bit puzzled ...

Reply:
See here: https://gul14.info/group__traits__h.html#ga28183aba128308ee3c88345edb3900b6

It is the same as [std::remove_cvref](https://en.cppreference.com/w/cpp/types/remove_cvref) and basically just removes `const`, `volatile`, and reference qualifiers `&`/`&&` from the given type.